### PR TITLE
Add Stanage, Remove ShARC & Remove purchasing HPC info

### LINKED ISSUE #717: Resources and Equipment  changes
+ [ ] Add Stanage to this page
+ [ ] Remove ShARC (Decomissioning 30th November 2023)
+ [ ] Remove? HPC purchasing Guidance sub-page and reference
  + This page is super super old (Pre 2019), and no longer possible to buy nodes (i.e. confusing for users, raised by Saul).

### PR DESCRIPTION
**Pull request by ptheywood**
+ Adds references to Stanage where appropraite
+ Removes references to ShARC from non-blog content where appropriate (Not all occurences replaced where unclear, i.e. #719)
+ Removes the very out of date HPC purchasing guide which is no-longer in line with current policy.

Closes #717

### CODE REVIEW

**Comment by ptheywood** [comment]
@willfurnass you don't have to review it, just wanted to make you aware the page was being murdered and that not mentioning purchasign hpc at all was the simpler option, rather than it going out of date again if policy changes.

File: pages/community/resources-and-equipment/languages/R.md [hunk 1]
```diff
@@ -14,9 +14,8 @@ R is a free, open source statistical computing language. It has become
 one of the dominant languages for statistics and data science.
 
   * The documentation for R on the University's High Performance Computing (HPC) clusters:
+      * Standard build on [Stanage](https://docs.hpc.shef.ac.uk/en/latest/stanage/software/apps/R.html)
```
Comment (willfurnass) [inline pages/community/resources-and-equipment/languages/R.md]: I'd describe the builds on Stanage and Bessemer as optimised builds as both were compiled on the system and link against optimised BLAS libs, as can be seen by running:

```
module load R
R -e 'sessionInfo()'
```

Comment (willfurnass) [reply pages/community/resources-and-equipment/languages/R.md]: Or just remove 'Standard'

**Review by willfurnass — APPROVED** [review]
LGTM.  Suggest omitting mention of NAG on ShARC too.

**Comment by ptheywood** [comment]
> LGTM. Suggest omitting mention of NAG on ShARC too.

I was going to leave the NAG/MAPLE links to deal with later, but yeah better to remove the refs for now and deal with if we want to keep the pages or not separately. 

I've also removed the `standard build on` segments as suggested

**Pull request merged**